e2e/nomostest/clean.go

```diff
@@ -606,3 +610,12 @@ func disableRootSyncDeletionPropagation(nt *NT) error {
 	}
 	return nil
 }
+
+func deleteTestAPIServices(nt *NT) error {
+	out, err := nt.Kubectl("delete", "apiservices", "-l", "testdata=true")
```

Comment:
Would have preferred this use the existing deleteObjects that uses the SDK, instead of shelling out to kubectl.